ts/src/test/static/request/hyperliquid.json

```diff
@@ -36,78 +36,82 @@
                 ],
                 "output": "{\"type\":\"metaAndAssetCtxs\"}"
             }
-    ],
```

Comment:
@ttodua I don't know what happened here, but this diff is a mess, do you think you can restore it, and add it again to show up only the added tests

Reply:
oh restored.